.circleci/config.yml

```diff
@@ -2,7 +2,11 @@ version: 2.1
 
 references:
   npm_cache_key: &npm_cache_key
-                   v1-dependency-npm-{{ checksum "package-lock.json" }}
+                   v1-dependency-npm1-{{ checksum "package-lock.json" }}
+  npm_cache_restore_key: &npm_cache_restore_key
+    keys:
+      - *npm_cache_key
+      - v1-dependency-npm1
   # Note: important that values of guids and keys have matching order!"
```

Comment:
The caching appeared to be broken: we were downloading all dependencies on each npm install.